crates/red_knot_python_semantic/src/python_version.rs

```diff
@@ -0,0 +1,136 @@
+use std::fmt;
+
+/// Enumeration of all supported Python versions
+///
+/// TODO: unify with the `PythonVersion` enum in the linter/formatter crates?
```

Comment:
Should this be `TargetVersion`?

```suggestion
/// TODO: unify with the `TargetVersion` enum in the linter/formatter crates?
```

Reply:
The equivalent enum in the linter/formatter crates is called `PythonVersion`, e.g. https://github.com/astral-sh/ruff/blob/b595346213c5343c27fc8ff152f15b85c7b7d44d/crates/ruff_linter/src/settings/types.rs#L45, and that's what this comment is referring to